IO/CMakeLists.txt

```diff
@@ -6,10 +6,7 @@ set(IO_SOURCES Keyboard.cpp
 set(IO_HEADERS Keyboard.h
                Mouse.h)
 
-if(WIN32)
-else()
-include_directories(${SDL_INCLUDE_DIRS})
-endif()
+include_directories(${SDL_INCLUDE_DIR})
```

Comment:
could you move that to sdl2.cmake  so this is available project-wide?

Reply:
done